docs/reference/devtools.md

```diff
@@ -0,0 +1,63 @@
+---
+title: devtools
+description:
+nav: 205
+---
+
+# devtools
+
+`devtools` middleware lets you use [Redux DevTools Extension](https://github.com/zalmoxisus/redux-devtools-extension)
+without Redux. Read more about the benefits of using [Redux DevTools for debugging](https://redux.js.org/style-guide/#use-the-redux-devtools-extension-for-debugging).
+
+```js
+devtools(stateCreatorFn, devtoolsOptions)
+```
+
+- [Reference](#reference)
+  - [Signature](#devtools-signature)
+- [Usage](#usage)
+- [Troubleshooting](#troubleshooting)
+  - [Only one store is displayed](#only-one-store-is-displayed)
+  - [Action names are labeled as 'anonymous'](#all-action-names-are-labeled-as-anonymous)
+
+## Reference
+
+### `devtools` Signature
+
+```ts
+devtools<T>(stateCreatorFn: StateCreator<T, [], []>, devtoolsOptions?: DevtoolsOptions): StateCreator<T, [], []>
+```
+
```

Comment:
Hi, I don't know if it would be useful to add that to create a slice in typescript and use devtools, I needed to add ` [['zustand/devtools', never]]` to the type, if not, TS marks an error when you try to set a name to the "action" (the third argument on `set` function) here and example:
```javascript
export const createPokemonSlice: StateCreator<
  PokemonStoreSlice,
  [['zustand/devtools', never]]
> = (set, get) => ({
  pokemons: [],
  selectedPokemon: {} as Pokemon,
  setPokemons: async () => {
    const response = await fetch('https://dummyapi.online/api/pokemon');
    const pokemonList = await response.json();
    const pokemons = pokemonList.slice(0, 5);
    set(() => ({ pokemons }), false, 'setPokemons');
  },
  setSelectedPokemon: (id) => {
    const selectedPokemon = get().pokemons.find((pokemon) => pokemon.id === id);
    set(() => ({ selectedPokemon }), false, 'setSelectedPokemon');
  },
});

```

Reply:
@Poncho0794 that would be part of usage section